docs/content/docs/guide/running.md

```diff
@@ -17,7 +17,7 @@ with that PipelineRun, and it will run if the following conditions are met:
 
   - The author is the owner of the repository.
   - The author is a collaborator on the repository.
-  - The author is a public member on the organization of the repository.
+  - The author is a member (public or private) on the repository's organization.
```

Comment:
Should we cover these scenarios in e2e :thinking:

Reply:
it  needs a lot of pre setup a github test org and another github user that is in public and another github user that it's in private.... a token generated to both and a lifecycle to manage those token rotations, and a bunch of other env variables to control this from the e2e tests...

i will be happy if someone else do and manage this :)

Reply:
Ya agree it requires some setup